dds/DCPS/transport/rtps_udp/RtpsUdpSendStrategy.cpp

```diff
@@ -281,6 +282,11 @@ RtpsUdpSendStrategy::send_single_i(const iovec iov[], int n,
 {
   OPENDDS_ASSERT(addr != ACE_INET_Addr());
 
+  if (addr == link_->transport().config().rtps_relay_address()) {
+    ++link_->transport().relay_rtps_send_count_;
+    link_->transport().report_relay();
```

Comment:
This seems like a pretty low level place to add a reporting method... should this maybe get called by a periodic task?

Reply:
Probably.  This was quick and dirty when I added it.  I'll remove it from this PR.